From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ludovic Henry <git@ludovic.dev>
Date: Tue, 13 Oct 2026 18:14:17 +0000
Subject: [PATCH 1/3] set the release version Screwdriver stamps at publish
 time

Upstream-Status: Inappropriate [upstream's Screwdriver generate_version step rewrites setup.cfg's version to 6.2.<build number> at release time and never commits it]

redislite 6.2.912183 on PyPI was cut from commit 802f8a0 ("Upgrade redis
to 6.2.14 (#190)"), byte-identical to the released sdist apart from
setup.cfg's version field. No git tag carries that version, so stamp it
the way upstream's release pipeline does.

Signed-off-by: Ludovic Henry <git@ludovic.dev>
---
 setup.cfg | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.cfg b/setup.cfg
index 65759b4..105c8a7 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -35,7 +35,7 @@ project_urls =
     Documentation = https://redislite.readthedocs.io/en/latest/
     Source = https://github.com/yahoo/redislite
 url = https://github.com/yahoo/redislite
-version = 6.2.14
+version = 6.2.912183
 
 [options]
 install_requires=
-- 
2.43.0

